tests/network/hotplug.go

```diff
@@ -238,6 +238,28 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
+			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
+			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
+
+			hotPluggedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(func(g Gomega) {
+				vmIfaceSpec := vmispec.LookupInterfaceByName(hotPluggedVM.Spec.Template.Spec.Domain.Devices.Interfaces, ifaceName)
+				g.Expect(vmIfaceSpec).NotTo(BeNil(), "VM spec should contain the new interface")
+
+				vmiIfaceStatus := vmispec.LookupInterfaceStatusByName(hotPluggedVMI.Status.Interfaces, ifaceName)
```

Comment:
`hotPluggedVMI` is not re-fetched in the `Eventually` per what I see, so it makes this retry pointless. Or am I missing something?